src/main/java/com/teragrep/aer_02/EventDataConsumer.java

```diff
@@ -146,7 +146,7 @@ public Long getValue() {
         SyslogMessage syslogMessage = new SyslogMessage()
                 .withSeverity(Severity.INFORMATIONAL)
                 .withFacility(Facility.LOCAL0)
-                .withTimestamp(enqueuedTime.toInstant())
+                .withTimestamp(enqueuedTime == null ? Instant.now() : enqueuedTime.toInstant())
```

Comment:
please add structured data indicating that the timestamp was sourced from this component i.e. aer_02@48577 timestamp_source=generated/time enqueued

Reply:
added